lib/menu.rb

```diff
@@ -0,0 +1,18 @@
+class Menu
+
+
+  attr_reader :foods
+
+  def initialize
+    @foods = { "pizza" => 7.99,
+              "pasta" => 7.50,
+              "steak" => 11.99,
+              "pint" => 3.99 }
+  end
+
+  def read_menu
+    p "Here is the menu for our restaurant"
```

Comment:
I understand why you did it in this instance, working up an interface when you're dealing with the terminal is not ideal. Of course, now you have web skills, I'm sure you can think of plenty of ways you could bring this to a more modern display device!